packages/discovery/src/discovery/DiscoveryLogger.ts

```diff
@@ -103,6 +103,11 @@ export class DiscoveryLogger {
       )}`,
     )
   }
+
+  logFetchingEvents(fromBlock: number, toBlock: number): void {
```

Comment:
Do we want to commit this to the main repo? If we want to present some kind of progress to the user of how many events were downloaded, then maybe look into some progress bar? Also maybe some rough time estimate calculated as `blocksToFetch * runningAvg(responseTimes)`

Reply:
Progress bar is flaky in logs, so for now I'm happy with this kind of logs:

```
Analyzing 0x1390f521A79BaBE99b69B37154D63D431da27A07
  Fetching events in range 15114701 - 15119999
  Fetching events in range 15120000 - 15129999
  Fetching events in range 15130000 - 15139999
  Fetching events in range 15140000 - 15149999
  Fetching events in range 15150000 - 15159999
```

Regarding ETA, I don't want to block this story further since it's urgent.